transactionoutbox-core/src/main/java/com/gruelbox/transactionoutbox/TransactionOutbox.java

```diff
@@ -102,6 +102,15 @@ default boolean flush() {
    */
   boolean flush(Executor executor);
 
+  /**
+   * Flushes a specific topic
+   *
+   * @param executor
+   * @param topicName
+   * @return
+   */
+  boolean flushTopic(Executor executor, String topicName);
```

Comment:
Why not allow a collection of topic names?

Perhaps two overloads
```
default boolean flushTopics(Executor executor, String... topicNames) {
  return flushTopics(executor, Arrays.asList(topicNames));
}
boolean flushTopics(Executor executor, Collectio<String> topicNames);
```

Reply:
> (can be extended to flush by topic(s) instead).
I can add it now though